pyproject.toml

```
@@ -1,59 +1,68 @@
-[tool.poetry]
+[project]
 name = "poetry"
 version = "2.0.0"
 description = "Python dependency management and packaging made easy."
-authors = ["Sébastien Eustace <sebastien@eustace.io>"]
+requires-python = ">=3.9,<4.0"
+dependencies = [
+    "poetry-core (==2.0.0)",
+    "build (>=1.2.1,<2.0.0)",
+    "cachecontrol[filecache] (>=0.14.0,<0.15.0)",
+    "cleo (>=2.1.0,<3.0.0)",
+    "dulwich (>=0.22.6,<0.23.0)",
+    "fastjsonschema (>=2.18.0,<3.0.0)",
+    "importlib-metadata (>=4.4) ; python_version < '3.10'",
+    "installer (>=0.7.0,<0.8.0)",
+    "keyring (>=25.1.0,<26.0.0)",
+        # packaging uses calver, so version is unclamped
+    "packaging (>=24.0)",
+    "pkginfo (>=1.12,<2.0)",
+    "platformdirs (>=3.0.0,<5)",
+    "pyproject-hooks (>=1.0.0,<2.0.0)",
+    "requests (>=2.26,<3.0)",
+    "requests-toolbelt (>=1.0.0,<2.0.0)",
+    "shellingham (>=1.5,<2.0)",
+    "tomli (>=2.0.1,<3.0.0) ; python_version < '3.11'",
+    "tomlkit (>=0.11.4,<1.0.0)",
+        # trove-classifiers uses calver, so version is unclamped
+    "trove-classifiers (>=2022.5.19)",
+    "virtualenv (>=20.26.6,<21.0.0)",
+    "xattr (>=1.0.0,<2.0.0) ; sys_platform == 'darwin'",
+]
+authors = [
+    { name = "Sébastien Eustace", email = "sebastien@eustace.io" }
+]
 maintainers = [
-    "Arun Babu Neelicattu <arun.neelicattu@gmail.com>",
-    "Bjorn Neergaard <bjorn@neersighted.com>",
-    "Branch Vincent <branchevincent@gmail.com>",
-    "Randy Döring <radoering.poetry@gmail.com>",
-    "Steph Samson <hello@stephsamson.com>",
-    "finswimmer <finswimmer77@gmail.com>",
-    "Secrus <b.sokorski@gmail.com>",
-]
-license = "MIT"
+    { name = "Arun Babu Neelicattu", email = "arun.neelicattu@gmail.com" },
+    { name = "Bjorn Neergaard", email = "bjorn@neersighted.com" },
+    { name = "Branch Vincent", email = "branchevincent@gmail.com" },
+    { name = "Randy Döring", email = "radoering.poetry@gmail.com" },
+    { name = "Steph Samson", email = "hello@stephsamson.com" },
+    { name = "finswimmer", email = "finswimmer77@gmail.com" },
+    { name = "Secrus", email = "b.sokorski@gmail.com" },
```

```suggestion
    { name = "Bartosz Sokorski", email = "b.sokorski@gmail.com" },
```
just keeping it consistent with other projects :wink: